frontend/src/component/project/ProjectAccess/ProjectAccessAssign/ProjectAccessAssign.tsx

```diff
@@ -336,7 +372,11 @@ export const ProjectAccessAssign = ({
                                 }}
                                 filterOptions={(options, { inputValue }) =>
                                     options.filter((option: IAccessOption) => {
-                                        if (option.type === ENTITY_TYPE.USER) {
+                                        if (
+                                            option.type === ENTITY_TYPE.USER ||
+                                            option.type ===
+                                                ENTITY_TYPE.SERVICE_ACCOUNT
+                                        ) {
```

Comment:
Same suggestion, maybe you can encapsulate this logic in a method `isAccount()` inside `IAccessOption`

Reply:
Unsure if that would bring much more clarity, and if I understood correctly it would also mean having to introduce a new class or class-like implementing this method, but feel free to share a suggestion of what you have in mind.

I guess we could have an auxiliary method here `const isAccount = (option: IAccessOption) => option.type === ENTITY_TYPE.USER || option.type === ENTITY_TYPE.SERVICE_ACCOUNT` if you think it's clearer.

FYI: https://github.com/Unleash/unleash/pull/2918 changes this to check for `!== ENTITY_TYPE.GROUP` instead, which may help solve this by simplifying the check.